package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,813 @@
+   GLU                         OE1, OE2
+   HIS         ND1, NE2        ND1, NE2    presence of H determines if donor
+   HSD         ND1             NE2
+   HSE         NE2             ND1
+   HSP         ND1, NE2
+   LYS         NZ
+   MET                         SD          see e.g. [Gregoret1991]_
+   SER         OG              OG
+   THR         OG1             OG1
+   TRP         NE1
+   TYR         OH              OH
+   =========== ==============  =========== ====================================
+
+.. table:: Heavy atom types for GLYCAM06 force field.
+
+   =========== =========== ==================
+   element     donor       acceptor
+   =========== =========== ==================
+   N           N,NT,N3     N,NT
+   O           OH,OW       O,O2,OH,OS,OW,OY
+   S                       SM
+   =========== =========== ==================
+
+Donor and acceptor names for the CHARMM27 force field will also work for e.g.
+OPLS/AA (tested in Gromacs). Residue names in the table are for information
+only and are not taken into account when determining acceptors and donors.
+This can potentially lead to some ambiguity in the assignment of
+donors/acceptors for residues such as histidine or cytosine.
+
+For more information about the naming convention in GLYCAM06 have a look at the
+`Carbohydrate Naming Convention in Glycam`_.
+
+.. _`Carbohydrate Naming Convention in Glycam`:
+   http://glycam.ccrc.uga.edu/documents/FutureNomenclature.htm
+
+The lists of donor and acceptor names can be extended by providing lists of
+atom names in the `donors` and `acceptors` keywords to
+:class:`WaterBridgeAnalysis`. If the lists are entirely inappropriate
+(e.g. when analysing simulations done with a force field that uses very
+different atom names) then one should either use the value "other" for `forcefield`
+to set no default values, or derive a new class and set the default list oneself::
+
+ class WaterBridgeAnalysis_OtherFF(WaterBridgeAnalysis):
+       DEFAULT_DONORS = {"OtherFF": tuple(set([...]))}
+       DEFAULT_ACCEPTORS = {"OtherFF": tuple(set([...]))}
+
+Then simply use the new class instead of the parent class and call it with
+`forcefield` = "OtherFF". Please also consider to contribute the list of heavy
+atom names to MDAnalysis.
+
+.. rubric:: References
+
+.. [Gregoret1991] L.M. Gregoret, S.D. Rader, R.J. Fletterick, and
+   F.E. Cohen. Hydrogen bonds involving sulfur atoms in proteins. Proteins,
+   9(2):99–107, 1991. `10.1002/prot.340090204`_.
+
+.. _`10.1002/prot.340090204`: http://dx.doi.org/10.1002/prot.340090204
+
+
+How to perform WaterBridgeAnalysis
+-----------------------------------
+
+All water bridges between arginine and aspartic acid can be analysed with ::
+
+  import MDAnalysis
+  import MDAnalysis.analysis.hbonds
+
+  u = MDAnalysis.Universe('topology', 'trajectory')
+  w = MDAnalysis.analysis.hbonds.WaterBridgeAnalysis(u, 'resname ARG', 'resname ASP', distance=3.0, angle=120.0)
+  w.run()
+
+.. Note::
```

Comment:
Is this relevant at all?? Remove the note if this is not true.

Reply:
@orbeckst I have rewritten the update_selection 1 so it is no longer True now. Thank you for pointing out.